src/fusion_power_plant.cc

```diff
@@ -196,9 +202,13 @@ bool FusionPowerPlant::ReadyToOperate() {
   // determine required tritium storage inventory
   double required_storage_inventory = fuel_usage_mass + SequesteredTritiumGap();
```

Comment:
I'm trying to remember why we need `reserve_inventory` + `sequestered_inventory` to startup but only need `fuel_usage_mass + sequesteredTritiumGap` to keep operating.  That is, given some of our back & forth discussion of these variables, why is startup based on `reserve_inventory` and continued operation based only on `fuel_usage_mass`?

Reply:
More generally, `SequesteredTritiumGap()` will be the same as `sequestered_equilibrium` before startup.  As such, there may be a path to simplify all of this and not need to store all of `startup_inventory`, `reserve_inventory`, and `fuel_usage_mass`.

Reply:
The bigger issue was that reserve_inventory was never considered when starting up. Reserve inventory is (supposedly) necessary to start up because plants want to have some certainty that something in their tritium recovery system isn't going to go wrong and cause them to suddenly shut down. However, once the plant is already operating, it doesn't make sense to stop operating simply because they had to dip into their reserves (that's the whole point of the reserve). After startup they only check that they have enough to run for a full time step, but the whole time they're trying to buy back up to their reserve again. I've put this as a discussion point for today's fuel cycle meeting so we can have a more in-depth conversation about this if needed.